src/js/utils/prop-types.js

```diff
@@ -214,6 +214,36 @@ export const roundPropType = PropTypes.oneOfType([
       PropTypes.string,
     ]),
   }),
+  PropTypes.shape({
+    corners: PropTypes.arrayOf(
+      PropTypes.oneOf([
+        PropTypes.shape({
+          'top-left': PropTypes.oneOfType([
+            PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
+            PropTypes.string,
+          ]),
+        }),
+        PropTypes.shape({
+          'top-right': PropTypes.oneOfType([
+            PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
+            PropTypes.string,
+          ]),
+        }),
+        PropTypes.shape({
+          'bottom-left': PropTypes.oneOfType([
+            PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
+            PropTypes.string,
+          ]),
+        }),
+        PropTypes.shape({
+          'bottom-right': PropTypes.oneOfType([
+            PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
+            PropTypes.string,
+          ]),
+        }),
+      ]),
+    ),
+  }),
```

Comment:
I think the propTypes don't match this:
```
round={{
            corners: {
              'top-left': 'small',
              'top-right': 'large',
              'bottom-left': 'medium',
              'bottom-right': 'xsmall',
            },
}}
```
PropTypes should be a shape instead of an array and include 'top-left', 'top-right', 'bottom-left', and 'bottom-right'

Reply:
Ok, I'll take a look and change.